src/index.ts

```diff
@@ -1,5 +1,4 @@
-export { default as SPAPipelineStack } from "./stacks/spa_pipeline_stack";
-export * from "./stacks/spa_pipeline_stack";
+export * from "./spa";
```

Comment:
Noticed I've started to put the SPA related constructs into the `/spa` directory instead of the `/constructs` directory.  This wasn't merely the style change – I noticed that when used this as imported library, just by calling `import { SPAPipelineStack } from "rylabs/ry-cdk-tools"` it would pull in more dependencies than I really needed (aws-rds & aws-ec2).  That's why I added these categorical exports, so we can do something like this:

https://github.com/RYLabs/townhall-frontend/blob/master/infra/cdk.ts#L2

When I did that, it just made more sense to group these related constructs in this directory.  Having said that, if you're just doing a S3 SPA, you will also need to require the Cloudfront libs.  And if decide to move the SPA pipeline stacks into this directory, which seems to make sense to me, then that would mean now requiring CodePipeline libs. More thinking needs to be done for this.